### DISCUSSION TITLE
MicroBuild.Core

### DISCUSSION THREAD

**Discussion by BenAigan** [git-ecosystem/git-credential-manager#419] [Ideas]
Noticed that when building the code for Windows that it requires MicroBuild.Core but this has been unlisted, can we get this updated during the next release cycle, see https://www.nuget.org/packages/MicroBuild.Core/

**Comment by mjcheetham** [comment]
We have updated our usage of the older `MicroBuild.Core` package to its replacement, [`Microsoft.VisualStudioEng.MicroBuild.Core`](https://www.nuget.org/packages/Microsoft.VisualStudioEng.MicroBuild.Core/).

https://github.com/GitCredentialManager/git-credential-manager/blob/e2f291b7bc81ad635c2c104925481d3f47e9258e/src/windows/Payload.Windows/Payload.Windows.csproj#L18
